dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -295,9 +291,6 @@ control dash_ingress(inout headers_t hdr,
 
         if (meta.dropped) {
             drop_action();
-        } else {
-            /* Send packet to port 1 by default if we reached the end of pipeline */
-            standard_metadata.egress_spec = 1;
```

Comment:
Would be nice to loop it back (`standard_metadata.egress_spec = stanard_metadata.ingress_port`). Deleting this assignment will send all packets to egress_spec=0 by default in bmv2.

Reply:
@chrispsommers yes it is already looped back by default at the beginning of the pipeline at line dash_pipeline.c : 232. I just removed this line so that it takes effect.

Reply:
@mukeshmv perfect, I missed that. Thanks!